src/main/java/de/zalando/aruha/nakadi/controller/TopicsController.java

```diff
@@ -72,8 +42,8 @@
     public ResponseEntity<?> listPartitions(@PathVariable("topicId") final String topicId) {
         try {
             return ok().body(topicRepository.listPartitions(topicId));
-        } catch (final NakadiException e) {
-            return status(503).body(e.getProblemMessage());
+        } catch (final Exception e) {
```

Comment:
I fixed that now in a context of https://github.com/zalando/nakadi/pull/84 (that is base on this branch)